src/domain/accounts/index.types.d.ts

```diff
@@ -162,6 +163,7 @@ interface IAccountsRepository {
 
   findByUsername(username: Username): Promise<Account | RepositoryError>
   // listBusinessesForMap(): Promise<BusinessMapMarker[] | RepositoryError>
+  findByNpub(npub: `npub${string}`): Promise<Account | RepositoryError>
```

Comment:
missed one

Reply:
ahh, i was searching for `npub1`, this didn't show up, thanks!